scico/test/linop/xray/test_astra.py

```diff
@@ -122,6 +122,11 @@ def test_adjoint_typical_input(testobj):
     adjoint_test(A, x=x, rtol=get_tol())
 
 
+def test_fbp(testobj):
+    x = testobj.A.fbp(testobj.y)
+    assert np.sum(np.abs(x)) > 0.0
```

Comment:
Consider checking that `x` is reasonably close to `testobj.x`.

Reply:
That did occur to me, but I decided against it because it wasn't obvious how to choose a reasonable threshold for "reasonably close". But easy to add if you think it's nevertheless worthwhile.

Reply:
On further consideration, I think what I'd prefer is a comment to the effect of "checks for bug where `fbp` would return all zeros when because ASTRA CPU FBP algorithm was being called on a GPU system."

Reply:
Done. Please confirm approval.